app/globals.scss

```diff
@@ -291,22 +291,20 @@ footer ul svg:hover {
 }
 
 /* NAVBAR */
+/* Note, the below is not in Josh's CSS file, but required to avoid the navbar spilling in the X-axis on small devices */
 /* Below 360px, the Navigation component won't naturally fit, so start to scale down the text */
-/* Not In Josh CSS
 .brand-text {
   @media (max-width: 359.98px) {
     font-size: 5.25vw;
   }
 }
 
 /* Below 385px, the NavWithHamburger component won't naturally fit, so start to scale down the text */
-/*
 .hamburger-brand-text {
   @media (max-width: 384.98px) {
     font-size: 4vw;
   }
 }
-*/
```

Comment:
I don't think we're using the hamburger menu so this is probably obsolete unless that changes?

Reply:
Yeah, I've left the hamburger code there in case we want it in future so we can have more menu links, but we can comment this styling for now so it doesn't increase the CSS bundle size for clients unnecessarily.